src/transaction.js

```diff
@@ -4,12 +4,15 @@ const EthObject = require('./ethObject')
 class Transaction extends EthObject{
 
   static get fields(){ return [
+    'chainId',
     'nonce',
-    'gasPrice',
+    'maxPriorityFeePerGas',
+    'maxFeePerGas',
     'gasLimit',
     'to',
     'value',
     'data',
+    'accessList',
     'v',
     'r',
     's',
```

Comment:
I ran this code on proofs using eth mainnet which has a collection of legacy/version 1/version 2 blocks, it does recreate properly, can you give example where it would break?